next/schema-generator/definitions/zalozenie-komunitnej-zahrady.ts

```diff
@@ -0,0 +1,373 @@
+            ),
+          },
+        ),
+        textArea(
+          'suhlasKomunity',
+          { title: 'Súhlas miestnej komunity', required: true },
+          {
+            placeholder: 'Popíšte',
+            helptextHeader:
+              'Dôležitý aspekt legitimity je zapojenie okolitej komunity – organizačný tím by mal získať súhlas miestnych obyvateľov a obyvateliek, ktorí priestor v súčasnosti využívajú.',
+          },
+        ),
+        textArea(
+          'organizacnyTim',
+          {
+            title: 'Organizačný tím záhrady',
+            required: true,
+          },
+          { placeholder: 'Popíšte' },
+        ),
+        textArea(
+          'prevadzka',
+          {
+            title: 'Prevádzka a údržba záhrady',
+            required: true,
+          },
+          {
+            placeholder: 'Popíšte',
+            helptextHeader:
+              'Kto a akým spôsobom bude zabezpečovať prevádzku záhrady z hľadiska údržby zelene (napríklad kosenie) a starostlivosti o poriadok? Ako bude zabezpečený odpad? Ideálne je, napríklad, zaviazať sa ku zero-waste režimu a každý, kto vytvorí odpad bude zodpovedný za jeho likvidáciu.',
+          },
+        ),
+        textArea(
+          'inkluzivnost',
+          {
+            title: 'Inkluzívnosť projektu a režim komunitnej záhrady',
+            required: true,
+          },
+          {
+            placeholder: 'Popíšte',
+            helptextHeader:
+              'Ako zabezpečíte otvorenosť projektu, kto bude mať priamy či nepriamy úžitok z projektu, ako vyberiete, kto bude záhradkár? Ako sa vysporiadate so situáciou, ak budete mať väčší záujem o záhradkárčenie, než budete mať kapacitu nasýtiť?',
+          },
+        ),
+        textArea(
+          'financovanie',
+          {
+            title: 'Spôsob financovania záhrady',
+            required: true,
+          },
+          {
+            placeholder: 'Popíšte',
+            helptextHeader:
+              'Aké budete mať členské poplatky? Využijete na tvorbu záhrady granty? Aké zdroje už máte a aké plánujete získať? V prípade, že už máte (predbežný) rozpočet, nahrajte ho do poľa nižšie.',
+          },
+        ),
+      ],
+    ),
+    conditionalStep(
+      'podmienenePrilohyPredschvaleny',
+      {
+        type: 'object',
+        properties: {
+          pozemok: {
+            type: 'object',
+            properties: {
+              typPozemku: {
+                not: {
+                  const: 'iny',
+                },
+              },
+            },
+            required: [],
+          },
+        },
+        required: [],
+      },
+      { title: 'Prílohy', customHash: 'predschvaleny' },
```

Comment:
As I mentioned previously, I pushed the product to change this, but I would still like to see "prilohy" here, e.g. "prilohy-predschvaleny-pozemok".